backend/fms_core/services/experiment_run_info.py

```diff
@@ -0,0 +1,267 @@
+from typing import Iterable, List, TextIO, Union
+from dataclasses import asdict, dataclass
+from io import StringIO
+import json
+from fms_core.models import (
+    Biosample, 
+    DerivedSample, 
+    DerivedBySample,
+    ExperimentRun, 
+    Index,
+    Individual,
+    Instrument, 
+    Library,
+    ProcessMeasurement,
+    Project,
+    PropertyValue,
+    Protocol,
+    Sample, 
+    SampleLineage,
+)
+
+Obj_Id = Union[int, None]
+
+@dataclass
+class RunInfoSample:
+    sample_name: Union[str, None]
```

Comment:
The direct pooled sample name is not very informative for techdev. I would store this specific as "pool_name" which is true and leave us space to use sample_name to store the alias.